SA/design_concept/content/introduction/goals.typ

```diff
@@ -0,0 +1,17 @@
+#import "../../../acronyms.typ": *
+
+= Goals <goals>
+
+With VisualFP we want to design a tool that allows graphical development of
```

Comment:
The goal of this term paper is to design a tool, VisualFP, that allows the graphical development of
functional code, is block-based and extensible.
The target audience of VisualFP are students learning to program.
Additionally, VisualFP should offer support for experienced programmers
that want to view their code in a visual context.

The core functionality of VisualFP will be implemented in a
#ac("PoC") to proof the feasibility of the design.

The main part of this paper is to transparently document the design process that was used
to create VisualFP. This allows others to better understand
the decisions that have been made, which alternatives were considered, and
to evaluate for themselves whether they want to follow the same path.

Reply:
Applied